pkg/secrets/aws.go

```diff
@@ -0,0 +1,48 @@
+package secrets
+
+import (
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	// AWSSecretType represent the secret type for AWS credentials.
+	AWSSecretType string = "secrets.kanister.io/aws"
+
+	// AWSAccessKeyID is the key for AWS access key ID.
+	AWSAccessKeyID string = "AWS_ACCESS_KEY_ID"
```

Comment:
typically the keys names are lowercased.

Reply:
@tdmanv See my comment.